core/src/main/java/com/sequenceiq/cloudbreak/core/flow2/cluster/provision/ClusterCreationActions.java

```diff
@@ -270,6 +275,7 @@ protected Selectable createRequest(ClusterViewContext context) {
             protected void doExecute(ClusterViewContext context, ClusterProxyGatewayRegistrationSuccess payload, Map<Object, Object> variables) {
                 clusterCreationService.clusterInstallationFinished(context.getStack());
                 jobService.schedule(context.getStackId(), StackJobAdapter.class);
+                syncJobService.schedule(context.getStackId(), StructuredSynchronizerJobAdapter.class);
```

Comment:
I can't see where is the unschedule for these kind of jobs. Shouldn't we unschedule when customer terminte a cluster?